### PR TITLE
33 build the block scheduling modal UI

### LINKED ISSUE #33: Build the block scheduling Modal UI
### Acceptance Criteria

- Looks like the design (with improvements where required)
- The user should be able to select and unselect the days
- Users should be able to select any time they want.
- "Cancel" button closes the modal

Image

### PR DESCRIPTION
**Pull request by devmarye**
Resolves #33  

### What does this PR do?

- Build the Model UI for Scheduling

### Screenshot
Screenshot 2024-03-15 at 12 41 52 AM

### CODE REVIEW

File: src/components/ui/dialog.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ const DialogOverlay = React.forwardRef<
   <DialogPrimitive.Overlay
     ref={ref}
     className={cn(
-      "fixed inset-0 z-50 bg-background/80 backdrop-blur-[1px] data-[state=open]:animate-in data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=open]:fade-in-0",
+      "fixed inset-0 z-50bg-background/80 backdrop-blur-[1px] data-[state=open]:animate-in data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=open]:fade-in-0",
```
Comment (TheoOkafor) [inline src/components/ui/dialog.tsx:22]: Is this intentional?

**Review by TheoOkafor — APPROVED** [review]
Well done @brendagold. I left 2 minor comments

**Comment by iamstarcode** [comment]
Looks good

**Review by iamstarcode — APPROVED** [review]
Nice job mam

File: src/components/ui/dialog.tsx (hunk 1 above)
Comment (brendagold) [reply src/components/ui/dialog.tsx:22]: No. thanks

**Comment by brendagold** [comment]
Thanks

**Pull request merged**